test/integration/apiary.js

```diff
@@ -0,0 +1,199 @@
+// Tests CAP-38 endpoints against the mock API server:
+// https://ammmock.docs.apiary.io/
+
+// All endpoints from here are tested:
+// https://docs.google.com/document/d/1pXL8kr1a2vfYSap9T67R-g72B_WWbaE1YsLMa04OgoU/edit
+const MOCK_SERVER = "http://private-anon-a06e1b25a0-ammmock.apiary-mock.com";
+
+describe("tests the /liquidity_pools endpoint", function() {
+  const lpId = "0569b19c75d7ecadce50501fffad6fe8ba4652455df9e1cc96dc408141124dd5";
+
+  it("GET /", function(done) {
+    let server = new StellarSdk.Server(MOCK_SERVER, {allowHttp: true});
+
```

Comment:
💡  Instead of redeclaring the server under every `it` we could declare it only once under `describe`.

Also, using a constant here would guarantee none of the tests is modifying the server:

```suggestion
  const lpId = "0569b19c75d7ecadce50501fffad6fe8ba4652455df9e1cc96dc408141124dd5";
  const server = new StellarSdk.Server(MOCK_SERVER, {allowHttp: true});

  it("GET /", function(done) {

```

Reply:
Good point! One of those things you don't see as you code but is obvious when you look at the whole.

Reply:
Done in 73a3cadf989ff35fab41db61ff8ad0344b3234dc.